test/images/scripts/fedora-24.setup

```diff
@@ -98,6 +102,9 @@ docker build -t cockpit/base /var/tmp/cockpit-base
 # docker images that we need for integration testing
 /var/lib/testvm/docker-images.setup
 
+# allow nested virtualization
+rmmod kvm_intel && modprobe kvm_intel nested=1
```

Comment:
Added.
I'm moving the 'modprobe kvm modules' to the test itself since this step depends on the actual test machine it's running on.